Makefile

```diff
@@ -5,7 +5,11 @@ DISK?=$(shell ls $(ROOT_DIR)/build/elemental*.qcow2 2> /dev/null)
 DISKSIZE?=20G
 ISO?=$(shell ls $(ROOT_DIR)/build/elemental*.iso 2> /dev/null)
 FLAVOR?=green
+ifdef PLATFORM
+ARCH=$(subst linux/,,$(PLATFORM))
```

Comment:
I think ARCH is only used to name the resulting disk images, so I would be in favor of removing it completely in the future.

Reply:
Yes, it seems to be only to name the resulting files. I had my riscv64 images named `elemental-green.x86_64.raw` :smile: